src/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -346,21 +370,46 @@ public static void ToByteArray()
             Assert.Equal(new byte[] { 0xd5, 0x10, 0xa1, 0xa8, 0x49, 0xfc, 0xc5, 0x43, 0xbf, 0x46, 0x80, 0x2d, 0xb8, 0xf8, 0x43, 0xff }, s_testGuid.ToByteArray());
         }
 
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public static void TryWriteBytes(byte[] b, Guid destination)
+        {
+            byte[] bytes = new byte[16];
```

Comment:
nit: where the type is obvious from the right side, you can use `var` ie `var bytes = new byte[16];` which is more compact...